### DISCUSSION TITLE
Problem with Extech Manometer

### DISCUSSION THREAD

**Discussion by Mr-Hooton** [artisan-roaster-scope/artisan#1510] [Q&A]
We just updated from version 2.8.0 to 2.10.2. In doing so I’m not getting correct readings from our Extech hd700. It worked properly on the previous computer and build. I have the port correct and windows is saying the driver is working just fine. 

Artisan shows a low reading like 0.1 mbar for a few seconds the u.u then a high reading like 14 mbar. It cycles through these numbers with little correlation to the readings on the unit. 

The Extech software reads it correctly when I have tried that on its own. 

The error I get is:
Extech755pressure(): conversion error, 10 bytes received 

Any thoughts on getting this working again?

**Comment by Mr-Hooton** [comment]
I installed an older driver and it now works. So be advised the Extech HD755 set up does not work with most current driver

**Comment by roasterdave** [comment]
Glad to hear it is working.  For the benefit of others, would you provide specifics about the drivers you reference, new and old, including versions and dates? Also what OS you are running?  Thanks.

**Reply by Mr-Hooton** [reply]
Running on Windows 11 Home build 22631.3155

The driver in question is Silicon Labs CP210x USB to UART Bridge
Working version: 6.7.4.261 9/19/2016
Current version: 10.3.0 6/24/2023
